From bfb1e27ad0c7736d4db12a9901f8c73c5f7f4461 Mon Sep 17 00:00:00 2001
From: Techassi <git@techassi.dev>
Date: Tue, 16 Dec 2025 10:01:19 +0100
Subject: pin sax to 1.2.4

This is a similar issue to what we already discovered in #1315
and #1316. For more details, see the linked PRs.
---
 superset-frontend/package.json | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/superset-frontend/package.json b/superset-frontend/package.json
index bb2641a67..8271aece2 100644
--- a/superset-frontend/package.json
+++ b/superset-frontend/package.json
@@ -365,11 +365,12 @@
       "ansi-regex": "^4.1.1"
     },
     "puppeteer": "^22.4.1",
-    "@types/react": "^16.9.53"
+    "@types/react": "^16.9.53",
+    "sax": "1.2.4"
   },
   "readme": "ERROR: No README data found!",
   "scarfSettings": {
     "allowTopLevel": true
   },
   "_id": "superset@0.0.0-dev"
-}
+}
\ No newline at end of file
